### ISSUE TITLE
enhancement: support different Python releases

### ISSUE THREAD

**Issue by zhangyoufu** [jendas1/jupyter-notebook-quick-look#2]
try several different `list.launchPath` like `"/usr/bin/python"`, `"/usr/local/bin/python"`, `"/usr/local/bin/python3"`, until the html generated successfully

**Comment by zhangyoufu** [comment]
I use Python 2 for daily work, and don't want to install a bunch of packages for both Python 2 & 3.

**Comment by singularitti** [comment]
Agree. It could also be `/usr/bin/env python3` to make it more general.
